src/main/resources/reference.conf

```diff
@@ -7,8 +7,10 @@ gpalloc {
 
   #GCP quota is one CreateProject operation per second.
   #https://cloud.google.com/resource-manager/docs/limits
+  # 1 project per 288 seconds means a max of 300 new GPAlloc projects per day. This will prevent GPAlloc from
+  # consuming the entirety of the Deployment Manager daily write operation quota of 1000 writes per day
```

Comment:
it is also about how many projects are we creating concurrently and this keeps it at 1 or 2 at a time.